static_inventories/wingedhelix_cluster.yml

```diff
@@ -89,4 +92,5 @@ wingedhelix_cluster:
     docs:
     rsyslog:
     nfs_server:
+    smb_server:
```

Comment:
Perhaps it would be best if create at the same time the smb_server configuration for the  BB and CF.

Reply:
Lets first try this when VLAN900 is available. If this does not fly we'll need to redesign it, which would result in double work for BB and CF.